src/Compilers/CSharp/Portable/Generated/CSharp.Generated.g4

```diff
@@ -744,11 +744,11 @@ lambda_expression
   ;
 
 parenthesized_lambda_expression
-  : modifier* parameter_list '=>' (block | expression)
+  : attribute_list* modifier* parameter_list '=>' (block | expression)
   ;
 
 simple_lambda_expression
-  : modifier* parameter '=>' (block | expression)
+  : attribute_list* modifier* parameter '=>' (block | expression)
```

Comment:
It feels like, given the feedback we've received so far, we should be implementing this just on `parenthesized_lambda_expression` for now.

Reply:
Should such attribute lists be parsed for error recovery and then we give a diagnostic for this case later on?

Reply:
That's a fair point.

---
In reply to: [599108764](https://github.com/dotnet/roslyn/pull/52038#discussion_r599108764) [](ancestors = 599108764)

Reply:
I'm not exactly sure when we'd want to issue this error, @cyrusnajmabadi would this be an error we'd issue in syntax, or would that interfere with incremental reparsing?

---
In reply to: [599118439](https://github.com/dotnet/roslyn/pull/52038#discussion_r599118439) [](ancestors = 599118439,599108764)

Reply:
Added a PROTOTYPE comment to report a binding error.

---
In reply to: [599123537](https://github.com/dotnet/roslyn/pull/52038#discussion_r599123537) [](ancestors = 599123537,599118439,599108764)

Reply:
if you can parse it out, but then report durign binding, that would be preferable.